feature-libs/order/README.md

```diff
@@ -2,7 +2,7 @@
 
 Spartacus Order feature can be added to the existing Spartacus application by running `ng add @spartacus/order`. For more information about Spartacus schematics, visit the [official Spartacus schematics documentation page](https://sap.github.io/spartacus-docs/schematics/).
 
-To see enhanced UI in Order History and Order Details page, 
+To see enhanced UI in Order History and Order Details page in B2C storefront, 
```

Comment:
[QUESTION]
Won't our New MyAccount feature work in B2B storefronts? If yes, what are constraints that make it impossible?

Reply:
it works for B2B as well but business wants to advise customers to use it for B2C alone, and from March, we can inform B2B to use as well